core/templates/services/platform-feature.service.ts

```diff
@@ -323,5 +316,9 @@ export class PlatformFeatureService {
   }
 }
 
+export const platformFeatureInitFactory = (service: PlatformFeatureService) => {
+  return (): Promise<void> => service.initialize();
```

Comment:
What happens if the promise fails? Does it break the page? Does the failure get logged?

Reply:
It's guaranteed in the platform feature service that this promise will resolve even if there's network failure, in this cases feature flags will be defaulted to true and athe page is not broken. The failure is logged.